plugin.video.vrt.nu/resources/lib/vrtplayer/vrtapihelper.py

```diff
@@ -31,7 +31,7 @@ def get_tvshow_items(self, path):
         for tvshow in tvshows:
             metadata_creator = metadatacreator.MetadataCreator()
             metadata_creator.mediatype = 'tvshow'
-            metadata_creator.plot = tvshow['description']
+            metadata_creator.plot = BeautifulSoup(tvshow['description'], 'html.parser').text
```

Comment:
I was looking for HTMLParser / html.parser and the unescape function. Since at some point we would like to get rid of BeautifulSoup4 when we removed all scraping.

Reply:
I implemented it using unescape().